src/Microsoft.Health.Dicom.Api.UnitTests/Features/Audit/AuditLoggingFilterAttributeTests.cs

```diff
@@ -10,41 +10,34 @@
 using Microsoft.AspNetCore.Mvc.Controllers;
 using Microsoft.AspNetCore.Mvc.Filters;
 using Microsoft.AspNetCore.Routing;
-using Microsoft.Health.Api.Features.Audit;
-using Microsoft.Health.Core.Features.Security;
 using Microsoft.Health.Dicom.Api.UnitTests.Features.Filters;
-using NSubstitute;
 using Xunit;
 using DicomAudit = Microsoft.Health.Dicom.Api.Features.Audit;
 
 namespace Microsoft.Health.Dicom.Api.UnitTests.Features.Audit;
 
 public class AuditLoggingFilterAttributeTests
 {
-    private readonly IClaimsExtractor _claimsExtractor = Substitute.For<IClaimsExtractor>();
-    private readonly IAuditHelper _auditHelper = Substitute.For<IAuditHelper>();
-
     private readonly DicomAudit.AuditLoggingFilterAttribute _filter;
 
     private readonly HttpContext _httpContext = new DefaultHttpContext();
 
     public AuditLoggingFilterAttributeTests()
     {
-        _filter = new DicomAudit.AuditLoggingFilterAttribute(_claimsExtractor, _auditHelper);
+        _filter = new DicomAudit.AuditLoggingFilterAttribute();
     }
 
     [Fact]
     public void GivenChangeFeedController_WhenExecutingAction_ThenAuditLogShouldBeLogged()
     {
         var actionExecutingContext = new ActionExecutingContext(
-            new ActionContext(_httpContext, new RouteData(), new ControllerActionDescriptor() { DisplayName = "Executing ChangeFeed." }),
+            new ActionContext(_httpContext, new RouteData(),
```

Comment:
We recently removed the StyleCop analyzers, but we've generally kept to "all arguments on one line or all arguments on their own line." (AAOOLOAAOTOL)